rainforest/resources.go

```diff
@@ -0,0 +1,115 @@
+package rainforest
+
+import "strconv"
+
+// Resource interface is a standardized way of looking at the types returned from the resources
+// list API. It treats every resource as a pair of ID and description of entity under this ID.
+type Resource interface {
+	GetID() string
+	GetDescription() string
+}
+
+// Folder type represents a single folder returned by the API call for a list of folders
+type Folder struct {
+	ID    int    `json:"id"`
+	Title string `json:"title"`
+}
+
+// GetID returns the Folder ID
+func (f Folder) GetID() string {
+	return strconv.Itoa(f.ID)
+}
+
+// GetDescription returns the Folder name
+func (f Folder) GetDescription() string {
+	return f.Title
+}
+
+// GetFolders returns a slice of Folders (their names and IDs) which are available
+// for filtering RF tests.
+func (c *Client) GetFolders() ([]Folder, error) {
+	// Prepare request
+	req, err := c.NewRequest("GET", "folders.json?page_size=100", nil)
```

Comment:
No. Maybe we should try to get the next 100 as well if the first request returned 100 results.